spec/lib/meaning_cloud/topics_spec.rb

```diff
@@ -5,6 +5,14 @@
     MeaningCloud.configuration.key = nil
   end
 
+  it 'has a default topic types configuration' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.